app/jobs/invoices/create_pay_in_advance_charge_job.rb

```diff
@@ -2,6 +2,8 @@
 
 module Invoices
   class CreatePayInAdvanceChargeJob < ApplicationJob
+    include ConcurrencyThrottlable
```

Comment:
Same as above:

Do we need to restrict it to only 5 concurrent calls? If not we can remove this line.
Anrok doesn't have concurrency restrictions, only 10 requests per second per API key.

Reply:
removed